analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -748,6 +748,15 @@ def add_arguments_to_parser(parser):
                                     "the analysis. USE WISELY AND AT YOUR "
                                     "OWN RISK!")
 
+    checkers_opts.add_argument('--no-missing-checker-error',
+                               dest="no_missing_checker_error",
+                               action='store_true',
+                               required=False,
+                               default=argparse.SUPPRESS,
+                               help="Emit a warning instead of an error when "
```

Comment:
Maybe include the recommendation from the commit message here too, that it is not advised to use this feature.